dsb-integration/src/test/java/com.spectralogic.dsbrowser.integration/tasks/Ds3DeleteBucketTaskTest.java

```
@@ -0,0 +1,124 @@
+        client.close();
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        new JFXPanel();
+        Platform.runLater(() -> {
+            final SavedSession savedSession = new SavedSession(
+                    TEST_ENV_NAME,
+                    client.getConnectionDetails().getEndpoint(),
+                    "80",
+                    null,
+                    new SavedCredentials(
+                            client.getConnectionDetails().getCredentials().getClientId(),
+                            client.getConnectionDetails().getCredentials().getKey()),
+                    false,
+                    false);
+            session = new CreateConnectionTask().createConnection(
+                    SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
+            try {
+                HELPERS.ensureBucketExists(DELETE_BUCKET_TASK_TEST_BUCKET_NAME, envDataPolicyId);
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        });
+    }
+
+    @Test
+    public void deleteBucket() throws Exception {
+        final CountDownLatch latch = new CountDownLatch(1);
+        Platform.runLater(() -> {
+            try {
+                final Ds3DeleteBucketTask deleteBucketTask = new Ds3DeleteBucketTask(session.getClient(),
+                        DELETE_BUCKET_TASK_TEST_BUCKET_NAME);
+                workers.execute(deleteBucketTask);
+                deleteBucketTask.setOnSucceeded(event -> {
+                    successFlag = true;
+                    latch.countDown();
+                });
+                deleteBucketTask.setOnFailed(event -> {
+                    latch.countDown();
+                });
+                deleteBucketTask.setOnCancelled(event -> {
+                    latch.countDown();
+                });
+            } catch (final Exception e) {
+                e.printStackTrace();
```

If this is a legitimate error condition, then let it fall out to the junit runner instead of handling it here, and swallowing the exception.